ui/src/track-layout/track-layout-slice.ts

```diff
@@ -592,3 +628,102 @@ function getLayoutContext(
         return officialMainLayoutContext();
     }
 }
+
```

Comment:
Tästä eteenpäin käytännössä koodia siihen, että osataan tutkia onko valittua toolpanelin tabia syytä vaihtaa, ja jos on, niin mikä olisi sopiva tabi valittavaksi